pytoda/datasets/drug_affinity_dataset.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         protein_padding_length: int = None,
         protein_add_start_and_stop: bool = False,
         protein_augment_by_revert: bool = False,
```

Comment:
Perhaps it's better to make both "protein_augment_by_revert" and "protein_sequence_augment" a single optional arg?

is backward compatibility the motivation you're keeping both?